modules/widget_utils.js

```diff
@@ -31,6 +30,14 @@ exports.show = function() {
 exports.cleanup = function() {
   delete exports.autohide;
   delete Bangle.appRect;
+  if (exports.origSetLCDOverlay){
+    Bangle.setLCDOverlay = exports.origSetLCDOverlay;
+    Bangle.setLCDOverlay();
+  }
+  if (exports.cleanUpOverlay){
```

Comment:
I don't think this property has an effect, should it be something like this?
```javascript
if (exports.cleanUpOverlay && exports.origSetLCDOverlay) { ... }
```

Reply:
Yes, I think I meant `Bangle.setLCDOverlay();` to be guarded by that.